src/main/java/io/cloudtrust/keycloak/services/resource/api/admin/UsersResource.java

```diff
@@ -1,5 +1,6 @@
 package io.cloudtrust.keycloak.services.resource.api.admin;
 
+import io.cloudtrust.keycloak.representations.idm.UsersPageRepresentation;
 import io.cloudtrust.keycloak.services.NotImplementedException;
```

Comment:
You can remove this line: NotImplementedException is not used anymore

Reply:
Well fine, then I ran an "Optimize imports" on the whole thing, there were other ghost imports :-)

Reply:
There are lots of them in many projects ! ;-)